mithril-aggregator/src/dependency_injection/builder.rs

```diff
@@ -1062,8 +1062,10 @@ impl DependenciesBuilder {
             Some(1),
             self.get_logger().await?,
         ));
+        let block_range_root_retriever = self.get_transaction_repository().await?;
         let cardano_transactions_builder = Arc::new(CardanoTransactionsSignableBuilder::new(
             transactions_importer,
+            block_range_root_retriever,
```

Comment:
Just wondering whether it would be better not to create the variable `block_range_root_retriever`.
```suggestion
            self.get_transaction_repository().await?,
```

Reply:
What difference would it actually make?